app/schemas/devices.js

```diff
@@ -23,8 +23,42 @@ var update = {
   }
 };
 
+var create = {
+  thingd: joi
+    .object({
+      name: joi.string().required(),
+      modbusSlaveID: joi.number().min(0).max(255).required(),
+      modbusSlaveURL: joi.string().required(),
```

Comment:
What about using the [`joi.uri()`](https://hapi.dev/module/joi/api/?v=17.1.1#stringurioptions) feature to validate this field?

Reply:
Done!